protocol/messenger_communities_import_discord.go

```diff
@@ -382,6 +382,7 @@ func (m *Messenger) cleanUpImport(communityID string) {
 	if deleteErr != nil {
 		m.logger.Error("clean up failed, couldn't delete community messages", zap.Error(deleteErr))
 	}
+	m.config.messengerSignalsHandler.DiscordCommunityImportCleanedUp(communityID)
```

Comment:
Do we actually need a separate signal for this? There should be some signal for leaving the community. And since this is still `Messenger`, I believe we could reuse it 🤔

Reply:
Seems like `LeaveCommunity` doesn't send a signal, only returns a response.

Anyway, it does a lot more work than just deleting the community, so calling it instead of `DeleteCommunity` would run code that is useless.

I think a separate signal is good anyway, because that way, if we ever needed to do some more actions on the front end when that signal happens, we can.

Reply:
ok, no prob 🙂